test/charlatan_http_request_test.dart

```diff
@@ -52,10 +52,7 @@ void main() {
         requestOptions: requestOptions,
       );
 
-      expect(subject.headers, {
-        ...headers,
-        'content-type': 'application/json; charset=utf-8',
-      });
+      expect(subject.headers, {...headers});
```

Comment:
Breaking change: `content-type` no longer added by default by `Dio`. Since this wasn't the point of the test, simply removing.